### PR TITLE
Add Mobile and Desktop Versions with Interactivity

### PR DESCRIPTION
**Pull request by AyoubM490**
###In this milestone, I:
- Added the Mobile Version of the about Page
- Added the Mobile Version of the Home Page
- Added the Desktop Version of the Home Page
- Added the Desktop Version of the About Page
- Added Interactivity of the menu with Javascript for the desktop and mobile versions
## video
[Short overview](https://www.loom.com/share/8bcada69e18b4195b7debd2e2d354ff1)

### CODE REVIEW

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @AyoubMs, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⛔❗

- [x] You have submitted a project, that does not have a descriptive README. Please consider adding a descriptive [README](https://github.com/microverseinc/readme-template) to your repository to make it look professional.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: about.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+              <li class="nav-link"><a href="#partner">Partner</a></li>
+              <li class="nav-link"><a href="index.html">Home</a></li>
+              <li class="nav-link">
+                <a class="tech" href="#mission">H.Tech</a>
+              </li>
+            </ul>
+            <div class="close">
+              <i class="fa fa-times"></i>
+            </div>
+          </nav>
```
Comment (ReshyResh) [inline about.html]: - [x] Currently, the hamburger icon is visible both on the mobile and desktop view. As per design, it's supposed to only be visible in the mobile view, while the desktop one has a wide navigation menu at the top of the page. Please make sure to use media queries and fix the views.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+                <div class="mission-title">
+                  <h2>Predictability</h2>
+                </div>
+                <div class="mission-detail">
+                  <p>
+                    Our devices know when they will be down, and they make you
+                    know beforehand!
+                  </p>
+                </div>
+              </div>
```
Comment (ReshyResh) [inline index.html]: - [x] This section is supposed to have elements displayed as rows on the desktop view and as columns on the mobile one. Currently, they are only being displayed as columns. Please make sure to address this issue, I suggest you also use Flexbox to position the content of the single card, not just the cards themselves. This way, you can choose whether to display as rows or columns both the cards and their own content.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+      <section id="partner" class="our-partners">
+        <h2 class="about-title">Partner</h2>
+        <ul class="partner-list flex-center">
+          <li><img src="img/logo1.png" alt="partner image" /></li>
+          <li><img src="img/logo2.png" alt="partner image" /></li>
+          <li><img src="img/logo3.png" alt="partner image" /></li>
+          <li><img src="img/logo4.png" alt="partner image" /></li>
+          <li><img src="img/logo5.png" alt="partner image" /></li>
+        </ul>
+      </section>
```
Comment (ReshyResh) [inline index.html]: - [x] This section also has different views for mobile and desktop screens. On desktop, it's supposed to be two elements side by side on each row, while on mobile they're all displayed in column. Please make sure to follow the design guidelines and make the required changes.

**Review by ReshyResh — CHANGES_REQUESTED** [review]
# Hi @AyoubMs , 

### Good job so far!
  - ✔ Used correct semantic tags for different sections of the webpage.
  - ✔ Elements are laid on the page using Flexbox.
  - ✔ Featured speakers section is dynamic.


#### There are some issues that you still need to work on to go tofinish your capstone but you are almost there!


## Required Changes ♻️

- [x] As per requirement, please record a video of yourself showcasing your project with a maximum length of 5 minutes. You can use Loom for it.
- [x] Please include a link to the live deployment of your project on Github Projects. The current deployment links to the Readme file.
- [x] Your project currently only has the mobile view, it is required to make use of media queries to have different views for desktop and mobile screens, please make sure to follow the design guidelines and add a desktop view.
- [x] Kindly edit the current Pull reuqest and add a more descriptive message of what has been done in the project, including links to the presentation video and live demo.

_Please check the comments under the review for more required changes._

----

Cheers and Happy coding! 👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 1 above)
Comment (AyoubM490) [reply about.html]: Done

File: index.html (hunk 2 above)
Comment (AyoubM490) [reply index.html]: Done

File: index.html (hunk 3 above)
Comment (AyoubM490) [reply index.html]: Done

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @AyoubMs, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [x] Please make sure that the video contains a highlight about a piece of the code you find interesting or are proud of making.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @AyoubMs , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- [x] Nice job done with highlighting the interesting piece of code of your project 👍 You have recorded a video with all proud parts of code and you concentrated only on this mentioned suggestion whole video. That is why you forgot about other requirements for video. Your previous video is very nice, but it is exactly without the suggested part of highlighting the interesting piece of code. So, to be done with the video, you have to record the video again like the first one, with added into it a few sentences about interesting/proud of part of the code.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elmar8287 — APPROVED** [review]
Hi @AyoubMs, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**